test/functional/apps/discover/group5/_shared_links.ts

```diff
@@ -74,7 +74,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
             },
             sort: [['@timestamp', 'desc']],
             columns: [],
-            index: 'logstash-*',
```

Comment:
Are we planning to introduce `dataSource` for shared links separately?

Reply:
Agree, it's not required for this PR. At some point `dataViewId`/`dataViewSpec`/`query` would not be enough to restore the state (for example, once we start supporting integrations) - same as with `index`/`query` in URL prior to this PR.